src/Libraries/DSIronPython/pkg.json

```diff
@@ -0,0 +1,19 @@
+{
+    "license": "",
+    "file_hash": null,
+    "name": "DynamoIronPython2.7",
+    "version": "2.5.0",
+    "description": "*** This is the official version of the IronPython Extension which will load the IronPython2 evaluation engine for Dynamo. *** \r\n\r\nIf you see the following warning 'The configured Python engine could not be found' when you try to run a Python Script node which is set to use IronPython2, the host application for Dynamo (Such as Revit or Civil 3d) will have chosen to exclude IronPython2 as default installed content and you will need to download this extension in order to enable IronPython2 again.\r\n\r\nYou have the ability to either download this IronPython2 Extension or use the alternative out-of-the-box CPython3 engine and migrate your legacy IronPython2 code to CPython3. While there are differences in language the Python basis is the same and we have provided a Migration Assistant to help you migrate your code to Python3.",
+    "group": "",
+    "keywords": [ "python", "ironpython" ],
+    "dependencies": [],
+    "host_dependencies": [ "Revit", "Civil 3D", "Alias", "Advance Steel", "FormIt" ],
+    "contents": "",
+    "engine_version": "2.13.0",
+    "engine": "dynamo",
+    "engine_metadata": "This pacakge is compatible with Dynamo2.13.0 release or newer",
```

Comment:
is this a real property that will make it up to the package manager, or this is just a comment?

Reply:
I can move it to the description if you think it is valuable info ...

Reply:
Ok, any opinions on the wording ?
I was thinking 
```
-------------------------------------------------
DynamoIronPython2.7 compatibility matrix
-------------------------------------------------
IronPython version    |    Dynamo version      |
|            2.1.0              |        [2.13.0, 2.13.x]      |
|            2.4.0              |        >=2.14.0              |
|            2.5.0              |        >=2.13.0              |
--------------------------------------------------
```

Reply:
👍

Reply:
I am not sure how backwards compatible these versions are...
Should we try to find the minimum supported Dynamo versions ? 
Or is 2.13.0 good enough as a minimum supported version ?
Also should I add the rest of the versions too ? 1.0.0 and 2.0.0 ?
Also should I say something about the broken versions ? ...2.2.0 and 2.3.0 ?

Reply:
If you specify engine version to be `"engine_version": "2.13.0"`, doesn't it mean that the package is compatible with Dynamo version >= 2.13.0?

Reply:
I believe the only thing it really means, is that the package was published from that version of Dynamo.

Reply:
We do show a warning in Dynamo if the pkg engine_vers is greater than Dynamo's version
https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs#L812

Reply:
also there is no good way to show the compatibility matrix as I have created it here. The description is just string from a JSON. I would have to use a simpler way to get our point across

Reply:
can't the string in json contain escape characters?
https://ozh.github.io/ascii-tables/

Reply:
they can...but the html side of things will need some work to be able to interpret those escaped characters. 
I think the style of some of the tags needs to change

Reply:
This is taking too much time. I propose that I publish the new package with just the engine property updated to reflect the compatibility (that would be pkg.engine_version: "2.13.0").
I can later update the description only (I think there is a way to just update the metadata of a package for super users)

Reply:
@Amoursol @mjkkirschner do you agree ?

Reply:
@pinzart Agreed, let's do a partial fix now to get this clearer for our users, and do a holistic one afterwards.